src/modules/unit/components/UnitBrowser.js

```diff
@@ -17,25 +32,32 @@ import {
   getDefaultSportFilter,
 } from '../helpers';
 
-const ActionButton = ({action, icon, isActive}) =>
+const addressBarMarker = require('@assets/markers/location.svg');
```

Comment:
The icon is broken (at least for me locally), `import addressBarMarker from '@assets/markers/location.svg';` seems to work tho.